data_model/derive/src/lib.rs

```diff
@@ -1,66 +1,47 @@
 //! A crate containing various derive macros for `data_model`
+#![allow(clippy::std_instead_of_core)]
 
 use proc_macro::TokenStream;
 use syn::parse_macro_input;
 
 mod filter;
 mod id;
 
-/// A derive macro for `Identifiable` trait and id-based comparison traits. Currently supports derivations only for
-/// `IdBox` and structs from the `data_model` crate that don't have generic parameters.
+/// A derive macro for `Identifiable` trait which also automatically implements [`Ord`], [`Eq`],
+/// and [`Hash`] for the annotated struct by delegating to it's identifier field. Identifier
```

Comment:
`by delegating to` what do you mean by this? the verb `delegate` requires you to specify what it is you are delegating.

Reply:
I mean that equality comparison of the struct implementing `Identifiable` will be done by comparing identifiers

Reply:
`Delegate` is a known pattern in programming. Since this is developer documentation, I think we can keep it.